chart/values.yaml

```
@@ -349,6 +349,8 @@ csi:
       ctrl_loss_tmo: "1980"
       # Kato (keep alive timeout) in seconds
       keep_alive_tmo: ""
+      # Fallback to nvme connect over tcp if host is not rdma capable
+      nvme_connect_fallback: true
```

Maybe something like this?

```suggestion
      tcpFallback: true
```

---

> Maybe something like this?

I can change this but we'll have inconsistency then, with other existing fields. wdyt?

---

hmm can't remember why the names are snake case, perhaps in order to match the kernel parameter names?
In this case this is not a known kernel parameter name?
(maybe it had been better to name these as camelCase and add list kernel parameter name as a comment, my bad)

---

converted to camelCase for this new value.